Resources/views/layout.html.twig

```diff
@@ -20,7 +20,7 @@ file that was distributed with this source code.
             {% block sonata_page_breadcrumb %}
                 <div class="row">
                     {% if sonata_seo_context is not defined %}
-                        {% set sonata_seo_context = 'homepage' %}
+                        {% set sonata_seo_context = 'page' %}
```

Comment:
so, what could be the good way to get locale ? request_stack is implemented form symfony 2.4
does sonata page bundle 3.x can be used in older version ?